unskript-ctl/unskript_ctl_config_parser.py

```diff
@@ -133,16 +134,13 @@ def parse(self):
                 combined_script = ';'.join(filtered_scripts)
                 print(f'Job: {self.job_name} contains custom script: {combined_script}')
                 if combine_check_types_and_script or combine_check_names_and_script:
-                    full_command += f' --script "{combined_script}" {notify}'
+                    full_command += f' --script "{combined_script}" {info} {notify}'
```

Comment:
What happens, if you have check types, script and info, wont it show --info 2 times, as it got added in line 112. Can you tabulate combinations and their corresponding generated cli command in a table and put that in the PR

Reply:
## Here is the Full Table of 15 Combination. 

*  unskript-ctl.sh run --info 
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --name <NAME>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```


*  unskript-ctl.sh run check --type <TYPE>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run --script <SCRIPT>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unksirpt-ctl.sh run --script <SCRIPT> --info
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run --script "/usr/local/bin/lb_pvc.sh"  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```


*  unskript-ctl.sh run check --name <NAME> --info 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --type <TYPE> --info 
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py

```

*  unskript-ctl.sh run check --name <NAME> --script <SCRIPT>
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --script "/usr/local/bin/lb_pvc.sh"  
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --type <TYPE> --script <SCRIPT>
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s  --script "/usr/local/bin/lb_pvc.sh"  
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```
*  unskript-ctl.sh run check --name <NAME> check --type <TYPE> 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes ; /usr/local/bin/unskript-ctl.sh run check --type k8s 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```


*  unskript-ctl.sh run check --type <TYPE> check --name <NAME> --script <SCRIPT>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes ; /usr/local/bin/unskript-ctl.sh run check --type k8s  --script "/usr/local/bin/lb_pvc.sh"  
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --type <TYPE> --script <SCRIPT> --info 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s --info --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --name <NAME> --script <SCRIPT> --info 
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes --info --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py

```

*  unskript-ctl.sh run check --name <NAME> check --type <TYPE> --info  

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info; /usr/local/bin/unskript-ctl.sh run check --type k8s  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --name <NAME> check --type <TYPE> --script <SCRIPT> --info 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info; /usr/local/bin/unskript-ctl.sh run check --type k8s --info --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

Reply:
@jayasimha-raghavan-unskript this doesnt look right to me, --info is repeated in both the commands
```
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info; /usr/local/bin/unskript-ctl.sh run check --type k8s --info --script "/usr/local/bin/lb_pvc.sh" 

```
This is this case **unskript-ctl.sh run check --name check --type --script <SCRIPT> --info**

Reply:
Took care of both scenarios.

Reply:
Updated with report too